ApiFirst/ApiFirst/Controllers/CustomersController.cs

```diff
@@ -62,7 +61,7 @@ public IActionResult Delete(int id)
         [HttpPut("{id}")]
         public IActionResult Update(int id, Customer model)
         {
-            var result = _customerService.Update(id, model);
+            int result = _customerAppServices.Update(id, model);
```

Comment:
Prefira utilizar var a int.
```suggestion
           var result = _customerAppServices.Update(id, model);
```